airflow/providers/grpc/hooks/grpc.py

```diff
@@ -72,7 +72,7 @@ def get_conn(self) -> grpc.Channel:
             channel = grpc.insecure_channel(base_url)
         elif auth_type in {"SSL", "TLS"}:
             credential_file_name = self._get_field("credential_pem_file")
-            creds = grpc.ssl_channel_credentials(open(credential_file_name).read())
+            creds = grpc.ssl_channel_credentials(open(credential_file_name, "rb").read())
```

Comment:
```suggestion
            with open(credential_file_name, "rb") as credential_file:
                creds = grpc.ssl_channel_credentials(credential_file.read())
```
Files should be closed when you do not plan to use them anymore. Otherwise, these files will be open until the process exits.

Reply:
I hadn't planned on changing anything but the mode of the read, but I'll commit your suggestion.